sale_partner_incoterm/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
@i-vyshnevska no need to `coding` remove all py file

Reply:
thanks, updated